src/op_mode/dhcp.py

```diff
@@ -264,8 +264,9 @@ def show_pool_statistics(raw: bool, family: ArgFamily, pool: typing.Optional[str
 def show_server_leases(raw: bool, family: ArgFamily, pool: typing.Optional[str],
                        sorted: typing.Optional[str], state: typing.Optional[ArgState]):
     # if dhcp server is down, inactive leases may still be shown as active, so warn the user.
-    if not is_systemd_service_running('isc-dhcp-server.service'):
-        Warning('DHCP server is configured but not started. Data may be stale.')
+    v = '6' if family == 'inet6' else ''
```

Comment:
Maybe change it to `v6` rather than just `6` so that the message reads `DHCPv6 server`?

Reply:
@dmbaturin  systemd service is `isc-dhcp-server6` without `v`

Reply:
@dmbaturin  This  commit fixes more systemd unit than the Warning message
So there was a check only for unit `isc-dhcp-server.service` but for not `isc-dhcp-server6.service`, so the service `isc-dhcp-serverv6.service` will be incorrect
I'm not against the warning with `v6` :)

Reply:
I have changed the warning message to include both DHCP/DHCPv6 .